devtools_m2/cypress/integration/CustomerCartInteractions/steps.js

```diff
@@ -136,6 +139,7 @@ Then('A simple cart event should be sent to Drip', function() {
     expect(item.quantity).to.eq(1)
     expect(item.total).to.eq(11.22)
   })
+  
```

Comment:
I really want to add a proper linter to these files. Pending that, could this newline be deleted?